lncfg/db.go

```diff
@@ -89,9 +89,7 @@ func (db *DB) GetBackends(ctx context.Context, dbPath string,
 			)
 		} else {
 			// Prefix will separate key/values in the db.
-			remoteDB, err = kvdb.GetEtcdBackend(
-				ctx, networkName, db.Etcd,
-			)
+			remoteDB, err = kvdb.GetEtcdBackend(ctx, db.Etcd)
```

Comment:
Removed the comment. Network separation is still possible even using the same database. Instead of the prefix that was merged into the flattened bucket prefix we now use `etcd` namespaces which allow more fine grained access control.